recipes/graylog-forwarder/files/pre-uninstall

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+set -e
+
+stopping="false"
+
+case "$1" in
+	# DEB based systems
+	remove)
+		stopping="true"
+		;;
+	upgrade)
+		stopping="false"
+		;;
+	deconfigure|failed-upgrade)
+		;;
+	# RPM based systems
+	0)
+		# Removal
+		stopping="true"
+		;;
+	1)
+		# Upgrade
+		stopping="false"
+		;;
+	*)
+		echo "[ERROR] pre-uninstall script called with unknown argument: '$1'"
+		exit 1
+		;;
+esac
+
+if [ "$stopping" = "false" ]; then
+	# Nothing to stop, exit early.
+	exit 0
+fi
+
+# Order is important! Some systems might have both commands!
+if command -v systemctl >/dev/null; then
```

Comment:
Let's only keep support for systemctl here. The rest of the forwarder packaging doesn't support anything else.